src/library_pthread.js

```diff
@@ -404,6 +413,10 @@ var LibraryPThread = {
     {{{ makeSetValue('__num_logical_cores', 0, 'cores', 'i32') }}};
   },
 
+  emscripten_transfer_canvas: function() {
+    Module['canvasTransferRequested'] = true;
```

Comment:
is this module property meant to be settable directly as well? if so then should be documented on `site/source/docs/api_reference/module.rst`.